plugins/outputs/sql/README.md

```diff
@@ -106,6 +105,17 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ##  {TABLE} - tablename as a quoted identifier
   # table_exists_template = "SELECT 1 FROM {TABLE} LIMIT 1"
 
+  ## Table update template
+  ## Available template variables:
+  ##  {TABLE} - table name as a quoted identifier
+  ##  {COLUMNS} - column definitions (list of quoted identifiers and types)
+  # column_template = "ALTER TABLE {TABLE} ADD COLUMN IF NOT EXISTS {COLUMN}"
+
+  ## Table columns query template
+  ## Available template variables:
+  ##  {TABLE} - table name as a quoted identifier
+  # column_query_template = "SELECT column_name FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME={TABLE}"
```

Comment:
Is there any reason the user would change that? I mean if we know those templates, why should the user configure them?

Reply:
Depending on the database, the query might (does) look different. For example we had to modify sqlite_test.go (part of this PR), as SQLite does not support INFORMATION_SCHEMA.

Reply:
Right, but we know the query for each driver type don't we? So we could setup the correct query internally, taking away the burden from the user...

Reply:
Ok, will remove it and leave a comment for devs who want to add a driver to the plugin.